po/fr.po

```diff
@@ -1,5 +1,723 @@
+
+#: src/Layouts/Partials/FillItem.vala:193
+msgid "Remove fill color"
+msgstr "Supprimer la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:277
+msgid "Show fill color"
+msgstr "Afficher la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:281
+msgid "Hide fill color"
+msgstr "Masquer la couleur de remplissage"
+
+#: src/Layouts/Partials/FillsPanel.vala:54
+msgid "Fills"
+msgstr "Remplissages"
+
+#: src/Layouts/Partials/FillsPanel.vala:65
+msgid "Add fill color"
+msgstr "Ajouter une couleur de remplissage"
+
+#: src/Layouts/Partials/TransformPanel.vala:96
+msgid "X"
+msgstr "X"
+
+#: src/Layouts/Partials/TransformPanel.vala:96
+msgid "Horizontal position"
+msgstr "Position horizontale"
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Y"
+msgstr "Y"
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Vertical position"
+msgstr "Position verticale"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "W"
+msgstr "W"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "Width"
+msgstr "Largeur"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "H"
+msgstr "H"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "Height"
+msgstr "Hauteur"
+
+#: src/Layouts/Partials/TransformPanel.vala:107
+msgid "Lock Ratio"
+msgstr "Ratio de verrouillage"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "R"
+msgstr "R"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "Rotation degrees"
+msgstr "Degrés de rotation"
+
+#: src/Layouts/Partials/TransformPanel.vala:126
+msgid "Flip Horizontally"
+msgstr "Retournement horizontal"
+
+#: src/Layouts/Partials/TransformPanel.vala:137
+msgid "Flip Vertically"
+msgstr "Retournement vertical"
+
+#: src/Layouts/Partials/TransformPanel.vala:165
+msgid "Position"
+msgstr "Position"
+
+#: src/Layouts/Partials/TransformPanel.vala:169
+msgid "Size"
+msgstr "Taille"
+
+#: src/Layouts/Partials/TransformPanel.vala:178
+msgid "Opacity"
+msgstr "Opacité"
+
+#: src/Layouts/RightSideBar.vala:111
+msgid "Search Layer"
+msgstr "Rechercher un calque"
+
+#: src/Lib/Managers/ExportManager.vala:192
+#: src/Lib/Managers/ExportManager.vala:219
+msgid "Generating preview, please wait…"
+msgstr "Génération de l'aperçu, veuillez patienter…"
+
+#: src/Lib/Managers/ExportManager.vala:313
+#, c-format
+msgid "Untitled %i"
+msgstr "Sans titre %i"
+
+#: src/Lib/Managers/ExportManager.vala:484
+msgid "Exporting images…"
+msgstr "Exporter des images…"
+
+#: src/Partials/ExportWidget.vala:117
+#, c-format
+msgid "%i × %i px · %s"
+msgstr "%i × %i px · %s"
+
+#: src/Partials/ExportWidget.vala:124
+msgid "Fetching image size…"
+msgstr "Récupération de la taille de l'image…"
+
+#: src/Partials/ZoomButton.vala:49
+msgid "Zoom Out"
+msgstr "Dézoomer"
+
+#: src/Partials/ZoomButton.vala:55
+msgid "Reset Zoom"
+msgstr "Remettre à zoom à zéro"
+
+#: src/Partials/ZoomButton.vala:62
+msgid "Zoom In"
+msgstr "Zoomer"
+
+#: src/Partials/ZoomButton.vala:68
+msgid "Zoom"
+msgstr "Zoom"
```

Comment:
I need context to valid this one (poke @albfan)

Reply:
In the headerbar:
![image](https://user-images.githubusercontent.com/22885898/95779820-11b68200-0ccb-11eb-91b6-4058ba85ecb9.png)

Reply:
Ok thanks, it's the good translation :+1: